md/manage-files-using-upload-servlet-and-rest-api.md

```diff
@@ -1,58 +1,40 @@
 # Manage files using upload servlet and REST API
 
-In the Community, Teamwork, Efficiency, and Performance editions
-you can upload files by doing a multipart post request on any of the available servlets.
+In the Community, Teamwork, Efficiency, and Performance editions you can upload files by doing a multipart post request on any of the available servlets.
 It returns the name of the temporary uploaded file.
-This file name can be used to link those files with any REST resource.
+This file name can be used to link those files with any REST resources.
 
-Available servlet list:
+## Available servlets
 
-* `/portal/fileUpload`, supports any type of file
+* `/portal/fileUpload`, supports any type of files
 * `/portal/processUpload`, supports only `.bar` files
 * `/portal/organizationUpload`, supports only `.xml` files
 * `/portal/actorsUpload`, supports only `.xml` files
 * `/portal/imageUpload`, supports only `.png`, `.jpg`, `.gif`, `.jpeg`, `.bmp`, `.wbmp` or `.tga` files
+* `/portal/pageUpload`, supports only `.zip` files
+* `/portal/applicationsUpload`, supports only `.xml` files
 * `/portal/connectorImplementation`, supports only `.zip` files (not available in Community edition)
-* `/portal/pageUpload`, supports only `.zip` files (not available in Community edition)
 * `/portal/reportUpload`, supports any type of file (not available in Community edition)
 * `/portal/resourceUpload`, supports only `.jar` files (not available in Community edition)
 * `/portal/profilesUpload`, supports only `.xml` files (not available in Community edition)
-* `/portal/applicationsUpload`, supports only `.xml` files (not available in Community edition)
 
-### Example: link an icon to a new organisation group
+## Example
 
-1. Upload an image by using a multipart post request on the image upload servlet, `http://localhost:8080/bonita/portal/imageUpload`.
-2. The image is stored temporarily in Bonita Home, in `/client/tenants/`_`tenantId`_`/tmp`.
-3. The servlet returns the name of the temporary file.
-4. When you create the new group, link this image to the group by specifying this temporary file name
+You can refer to the [example-upload-servlet](https://github.com/Bonitasoft-Community/example-upload-sevlet) project that demonstrate the two use cases explained below.
 
-#### Example
-Request url
-/API/identity/group
 
-Request method
+### Link an icon to a new organization group
 
-POST
+1. Call to the login service: `/bonita/loginservice` for authentication.
```

Comment:
remove the "to" in order to be consistent with the other points (imperative form)